app/scripts/views/sign_up.js

```diff
@@ -35,6 +37,18 @@ function (Cocktail, _, p, BaseView, FormView, Template, AuthErrors, mailcheck,
     return null;
   }
 
+  var NO_OF_HASHING_FUNCTIONS = 8;
+  var bloom = null;
+  $.ajax({
+      method: 'GET',
+      dataType: 'json',
+      url: '../bower_components/fxa-bloom-filter/src/bloomdata_short_pwd.js',
+      error: function (jqxhr, status, err) { this.metrics.logEvent(status + ' ERROR ' + err); }
+    }).done(function (res){
+      bloomfilter = bloomfilter;
```

Comment:
The pre hook wouldnt let me push unless i added this in. Will remove(if someone can tell me how to bypass amdcheck).